### PR TITLE
Documentation: Adding further information about tests (e.g. skipping)

### PR DESCRIPTION
**Pull request by Corentin-Leffy**
Hello 👋

I add some more informations about tests result (like what does `~` means, etc...).

Thanks ☺️

Issue : #246

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ Choose your operating system:
...
+00:01 +4 ~3 -1: Some tests failed.
+```
+
+`+` describes the number of tests passed. In this example, 4 tests are passed.
+
+`~` describes the number of tests skipped. In this example, 3 tests are skipped.
+
+`-` describes the number of tests failed. In this case, there is only one test that failed.
+
+You should disable all of the skips before completing an exercise.
```
Comment (devkabiir) [inline docs/TESTS.md:27]:
```suggestion
It is advised to enable tests one by one as you implement the functionality being tested or as you move towards edge cases. You can do so by flipping the `skip: false` to `skip: true` flag for each test. You can also submit your exercise without passing all tests to get feedback.
```

File: docs/TESTS.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ Choose your operating system:
 * [Mac OS X](#mac-os-x)
 * [Linux](#linux)
 
+## Understanding Skips
```
Comment (devkabiir) [inline docs/TESTS.md:9]:
```suggestion
## Understanding Test Result
```

File: docs/TESTS.md [hunk 3]
```diff
@@ -6,6 +6,26 @@ Choose your operating system:
...
+
+The `skip` parameter instructs the test suite to not run a test. This is commonly used to avoid running tests of unimplemented functionality, so you can focus on the part you are currently working on.
```
Comment (devkabiir) [inline docs/TESTS.md:11]:
```suggestion
```

File: docs/TESTS.md [hunk 4]
```diff
@@ -6,6 +6,26 @@ Choose your operating system:
...
+❯ pub run test
+00:01 +4 ~3 -1: Some tests failed.
+```
+
+`+` describes the number of tests passed. In this example, 4 tests are passed.
+
+`~` describes the number of tests skipped. In this example, 3 tests are skipped.
+
+`-` describes the number of tests failed. In this case, there is only one test that failed.
+
```
Comment (devkabiir) [inline docs/TESTS.md:26]:
```suggestion

The `skip` parameter instructs the test suite to not run a test. This is commonly used to avoid running tests of unimplemented functionality, so you can focus on the part you are currently working on.

```

File: docs/TESTS.md [hunk 5]
```diff
@@ -6,6 +6,26 @@ Choose your operating system:
 * [Mac OS X](#mac-os-x)
 * [Linux](#linux)
 
+## Understanding Skips
+
+The `skip` parameter instructs the test suite to not run a test. This is commonly used to avoid running tests of unimplemented functionality, so you can focus on the part you are currently working on.
+
+The test run summary will list the number of skipped tests.
```
Comment (devkabiir) [inline docs/TESTS.md:13]:
```suggestion
The test run summary will list the number of passed, skipped and failed tests.
```

**Review by devkabiir — CHANGES_REQUESTED** [review]
Thank you for your contribution :rocket:

What are your thoughts on my suggestions?

File: docs/TESTS.md (hunk 1 above)
Comment (Corentin-Leffy) [reply docs/TESTS.md:27]: I like that change 👍

**Review by devkabiir — APPROVED** [review]
Thank you :rocket:
